src/test/ui/feature-gate-isa_attribute.rs

```diff
@@ -0,0 +1,5 @@
+#[instruction_set]
+//~^ ERROR the `#[instruction_set]` attribute is an experimental feature
```

Comment:
You need to make the error code part of the error annotation itself, right now it's looking for the literal error text 'error: E0658, E0778'. Probably this should be documented  better, all I found was https://rustc-dev-guide.rust-lang.org/tests/adding.html#use-same-error-line-as-defined-on-error-annotation-line-above.

```suggestion
//~^ ERROR the `#[instruction_set]` attribute is an experimental feature [E0658]
```

```
unexpected errors (from JSON output): [
    Error {
        line_num: 1,
        kind: Some(
            Error,
        ),
        msg: "1:1: 1:19: malformed `instruction_set` attribute input",
    },
    Error {
        line_num: 1,
        kind: Some(
            Error,
        ),
        msg: "1:1: 1:19: must specify an instruction set [E0778]",
    },
]

not found errors (from test file): [
    Error {
        line_num: 1,
        kind: Some(
            Error,
        ),
        msg: "E0658, E0778",
    },
]

failures:
    [ui] ui/feature-gate-isa_attribute.rs
```